styleguide/_themes/global/scss/fonts.scss

```diff
@@ -159,3 +172,24 @@ i {
 .copyright {
   font-size: .75em;
 }
+
+blockquote {
+  border-left: 0;
+  font-size: 1.75rem;
```

Comment:
Are we intentionally mixing `em` and `rem` units in this file (and throughout zoolander in general)? They definitely have semantic differences so it's possible it's intentional, but I would recommend trying to stick with a single unit for everything font related if possible.